ecl/hthor/hthor.ipp

```diff
@@ -188,7 +188,7 @@ static bool verifyFormatCrcSuper(unsigned helperCrc, IDistributedFile * df, bool
 }
 
 #define IMPLEMENT_SINKACTIVITY \
-    virtual unsigned queryOutputs() { return 0; } \
+    virtual unsigned queryOutputs() const override { return 0; } \
```

Comment:
Adding override to just one function causes problems with clang (especially apple)